lib/transaction/transaction_test.go

```diff
@@ -87,3 +88,18 @@ func TestIsWellFormedTransactionWithInvalidSignature(t *testing.T) {
 	err = tx.IsWellFormed(networkID)
 	require.NotNil(t, err)
 }
+
+func TestIsWellFormedTransactionMaxOperationsInTransaction(t *testing.T) {
+	var err error
+
+	MaxOperationsInTransaction := common.MaxOperationsInTransaction
+	defer func() {
+		common.MaxOperationsInTransaction = MaxOperationsInTransaction
+	}()
+	common.MaxOperationsInTransaction = 2
```

Comment:
We should not modify global state like this... It breaks abstraction big time and prevents the tests from running in parallel.

Either:
- We make 1001 operations to test
- We make it a parameter to the function

Reply:
@Geod24 Yes, you right. In parallel env, it makes problem.